src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.html

```diff
@@ -24,7 +24,7 @@
     @if (modelingExercise) {
         <jhi-resizeable-container [isExerciseParticipation]="true" [isBeingPrinted]="isPrinting" [expandProblemStatement]="expandProblemStatement">
             <!--region Left Panel-->
-            <div left-body class="submission-container d-flex flex-column w-100">
+            <div left-body class="submission-container d-flex flex-column w-100" [ngClass]="{ 'no-submission': submission && !submission.submitted && isExamSummary }">
```

Comment:
Is it possible to wrap this around a big`@if(submission && !submission.submitted && isExamSummary) {}` block?